packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -173,12 +173,15 @@ class MongoshNodeRepl implements EvaluationListener {
     instanceState.setEvaluationListener(this);
     await instanceState.fetchConnectionInfo();
 
-    let mongodVersion = instanceState.connectionInfo.buildInfo?.version;
-    const apiVersion = serviceProvider.getRawClient()?.serverApi?.version;
-    if (apiVersion) {
-      mongodVersion =
-        (mongodVersion ? mongodVersion + ' ' : '') +
-        `(API Version ${apiVersion})`;
+    const { buildInfo, extraInfo } = instanceState.connectionInfo;
+    let mongodVersion = extraInfo?.is_stream ? undefined : buildInfo?.version;
+    if (!extraInfo?.is_stream) {
```

Comment:
What happens when a user tries to connect to SPI with a stable API version defined?